src/main/scala/ai/privado/semantic/Language.scala

```diff
@@ -22,18 +22,10 @@
 
 package ai.privado.semantic
 
```

Comment:
@khemrajrathore did we create a new Language.scala as a subpackage inside java package?

Reply:
yes, we had this earlier outside, now we have segregated into 2, generic ones outside, lang specific inside java